assets/js/status.js

```diff
@@ -8,28 +8,32 @@ const avatarProvider = "{{ site.avatar_provider }}";
 
 setTimeout(function() {
     $.getJSON(api + host, function(json) {
-        let count = 0;
-        let max = 0;
-        let motd = "";
-        let playersArray = [];
-
         const online = json.online;
 
         if (online) {
-            const players = json.players.online;
-            max = json.players.max;
-            motd = json.motd.clean[1];
-            const mapName = motd.replace('» ', '').replace(' «', '').replace(/.§./gi, '');
-            const mapUrl = imageProvider + mapName + "/map.png";
-            count += players;
-            playersArray = json.players.list;
-            fetchPlayerCount(count + "<small>/" + max + " players</small>");
-            fetchMapName(mapName);
+            const pgm = json.bukkit_extra.pgm;
```

Comment:
If none of the plugins submits extra data, `bukkit_extra` object won't exist.
https://github.com/Electroid/SportPaper/commit/d85121f17590656a46750a405e824a2824b3e122#diff-1119feffcbb0ffa554c7f9e223e49cb5R100-R102